### DISCUSSION TITLE
LINE OUT sgtl5000

### DISCUSSION THREAD

**Discussion by glido1** [ElectroTechnique/electrotechnique.github.io#52] [Hardware Issues]
Hi.
I read that Volume tsynth only works with the headphone output.
Is it possible in tsynth code to switch sgtl5000 to line output?

![Bez naBez nazwyzwy](https://user-images.githubusercontent.com/82583604/118624329-5b1f0080-b7c9-11eb-80f2-0f546f3937eb.jpg)

Now I need the correct impedance at the analog output.

**Comment by Electrotechnique** [comment]
The volume control can be changed to alter `volumeMixer.gain(0, 1.0f);` This will work for all outputs including digital audio. Leave the line level alone in the code, it works in an odd way.

Is it possible for you to post here without swearing? For native English speakers it is jarringly aggressive.
